.github/workflows/iuf-container.yaml

```diff
@@ -34,14 +34,14 @@ jobs:
       contents: read
       id-token: write
     env:
-      DOCKER_REPO: artifactory.algol60.net/csm-docker/stable/iuf
-      DOCKER_TAG: v0.1.1
+      DOCKER_REPO: artifactory.algol60.net/csm-docker/unstable/iuf
```

Comment:
I'm guessing the changes in this file were left over from testing and will be removed before you merge.

Reply:
Reverted.